src/styles/blocks/card.css

```diff
@@ -0,0 +1,91 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin-top: 16px;
+  margin-right: 3px;
+  margin-left: 3px;
+  object-fit: cover;
+}
+
+.card__description-link {
+  display: block;
+  margin-top: 40px;
+  color: #060b35;
+  font-family: inherit;
```

Comment:
Unnecessary line. you only need to add `font-family` to `body` selector, and all other tags will include this font too. Check other cases.